lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -154,7 +155,7 @@
 <hr class="divider" />
 %endif
 
-<div class="member-lists-management membership-section">
+<div class="member-lists-management membership-section" aria-live="polite">
```

Comment:
This was added because of feedback => 2nd point in section membership  

https://github.com/edx/edx-platform/pull/9038#issuecomment-131098884